pkgs/racket-doc/scribblings/reference/module-reflect.scrbl

```diff
@@ -535,6 +535,48 @@ If @racket[provided] is @|void-const|, then the module is
 @tech{visit}ed but not @tech{instantiate}d (see @secref["mod-parse"]),
 and the result is @|void-const|.}
 
+@bold{Note}: because dynamic-require is a procedure, it is necessary to wrap @racket[mod]
+values matching @racket[module-path] grammar for @tech{require} with @tech{quote}
```

Comment:
I think explicitly suggesting `quote` isn't quite right. Instead, we want to emphasize that just what you put in the grammar for `require` won't work, and you need something that evaluates to an s-expression. `quote` is one way to do that.

Reply:
Also, instead of using `tech` here, use `@racket[require]` and `@racket[quote]`.

Reply:
Makes sense, after all we could have used arbitrary S-expression valued functions to generate the necessary grammar.

Reply:
Also, would you mind briefly explain the difference between `@tech` and `@racket` I was wondering about that while composing the initial draft.